MapboxNavigation/NavigationMapView.swift

```diff
@@ -1025,39 +1027,122 @@ extension Dictionary where Key == Int, Value: MGLStyleValue<NSNumber> {
 }
 
 //MARK: NavigationMapViewDelegate
-
+/**
+ The `NavigationMapViewDelegate` provides methods for configuring the NavigationMapView, as well as responding to events triggered by the NavigationMapView.
+ */
 @objc(MBNavigationMapViewDelegate)
 public protocol NavigationMapViewDelegate: class  {
+    /**
+     Asks the reciever to return an MGLStyleLayer for routes, given an identifier and source.
+     - note: This method is invoked at initalization time and anytime routes are added.
```

Comment:
Typo: “any time”. (“Anytime” is an adverb.)

This can be an ordinary paragraph; it doesn’t need to be called out specially in a `note`.